laterpay/views/frontend/partials/post/gift/gift-pass.php

```diff
@@ -22,9 +22,12 @@
 if ( 0 !== absint( $gift_pass['access_to'] ) ) {
     $access_dest = $category->name;
 }
+
+$pass_id = ( ! empty( $gift_pass['pass_id'] ) ) ? $gift_pass['pass_id'] : '';
+$pass_id = ( empty( $pass_id ) && ! empty( $gift_pass['id'] ) ) ? $gift_pass['id'] : '';
 ?>
 
-<div class="lp_js_giftCard lp_gift-card lp_gift-card-<?php echo esc_attr( $gift_pass['pass_id'] ); ?>">
+<div class="lp_js_giftCard lp_gift-card lp_gift-card-<?php echo esc_attr( $pass_id ); ?>">
```

Comment:
Should the card be displayed if `$pass_id` is empty in the end?

Reply:
Added condition. if `$pass_id` is empty don't render template.